cursos/helm/00_solucions/99_other_wp/actividade-1-charts-helm/wp-release/templates/secret.yaml

```diff
@@ -0,0 +1,8 @@
+# secret.yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: wp-secrets
+type: Opaque
+data:
+  root-password: QzBudHI0c2VuNA==
```

Comment:
The secret is not really encrypted, shouldn't be stored in the repo. You could use a service like aws parameter store, or sops to really encrypt the secret in the code.